repos/system_upgrade/el7toel8/actors/dosfstools/actor.py

```diff
@@ -0,0 +1,30 @@
+from leapp.actors import Actor
+from leapp.models import InstalledRedHatSignedRPM
+from leapp.tags import ChecksPhaseTag, IPUWorkflowTag
+from leapp.reporting import Report
+from leapp.libraries.common.reporting import report_with_remediation
+
+
+class Dosfstools(Actor):
```

Comment:
Could we have a more descriptive name? Maybe something like "CheckDosfstoolsInstalled"?